modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc

```diff
@@ -1,4 +1,5 @@
 = User Authentication with LDAP
+:experimental:
```

Comment:
this is an extra line that is not in the master PR ???

Reply:
Will remove.

Reply:
Done.